### PR TITLE
Ports the triple-bladed energy sword from April Fools

### PR DESCRIPTION
**Pull request by PKPenguin321**
trisword

An April Fools favorite has finally been ported to the real game. Same stats as last time:
- Can be purchased in the traitor uplink **<-- I'm considering making it require 3 eswords to build instead now that it's in the real game but I'm not sure**
- Requires 2 hands
- Deals a whopping 45 damage per hit
- Has 100% block chance for ALL projectiles
- Has a 80% block chance for any melee attacks
- Can be rapidly spun, causing it to shoot electricity that stuns and deals high burn (on a cooldown)

:cl: PKPenguin321
rscadd: The devious triple-bladed energy sword can now be purchased from the uplink.
/:cl:

### CODE REVIEW

**Comment by Jordie0608** [comment]
Considering what it does, should probably be assembled and not purchased.

**Comment by TechnoAlchemisto** [comment]
that block chance is pretty ridiculous

**Comment by optimumtact** [comment]
Just no, I wish april fools had never gone so far in the first place.

**Comment by optimumtact** [comment]
In fact the more I think about it, just no

**Comment by YakumoChen** [comment]
isn't this supposed to be a joke item? you can't actually pick it up, and wielding it makes you drop it...

File: code/game/objects/items/weapons/melee/energy.dm [hunk 1]
```diff
@@ -247,3 +247,66 @@
...
+	w_class = 2
+	icon_state = "trisaber_off"
+	item_state = "sword0"
+	var/progress = 0
+	var/on = 0
+
+/obj/item/weapon/trisword/attack_self(mob/living/carbon/user)
+	if(on || progress > 5)
+		return
+	if(progress == 0)
```
Comment (RemieRichards) [inline code/game/objects/items/weapons/melee/energy.dm]: use a ```switch()```

**Comment by YakumoChen** [comment]
optimum obviously didn't read the code, what an idiot

**Comment by optimumtact** [comment]
Well if you want to be like that then I can be salty too.

**Comment by ike709** [comment]
:tangerine: citrusy tyranny :tangerine:

**Pull request closed without merging**